config/samples/beat/filebeat_es_kibana_apm.yaml

```diff
@@ -0,0 +1,67 @@
+# This sample sets up an Elasticsearch cluster and a Kibana instance preconfigured for that cluster
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: elasticsearch-sample
+spec:
+  version: 7.7.0
+  nodeSets:
+  - name: default
+    count: 2
+    config:
+      # This setting could have performance implications for production clusters.
+      # See: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-virtual-memory.html
+      node.store.allow_mmap: false
+---
+apiVersion: kibana.k8s.elastic.co/v1
+kind: Kibana
+metadata:
+  name: kibana-sample
+spec:
+  version: 7.7.0
+  count: 1
+  elasticsearchRef:
+    name: elasticsearch-sample
+  podTemplate:
+    metadata:
+      labels:
+        foo: bar
+    spec:
+      containers:
+        - name: kibana
+          resources:
+            limits:
+              memory: 1Gi
+              cpu: 1
+---
+apiVersion: apm.k8s.elastic.co/v1
+kind: ApmServer
+metadata:
+  name: apm-sample
+spec:
+  version: 7.7.0
+  count: 1
+  elasticsearchRef:
+    name: elasticsearch-sample
+---
+apiVersion: beat.k8s.elastic.co/v1beta1
+kind: Beat
+metadata:
+  name: filebeat-sample
+spec:
+  type: filebeat
+  version: 7.7.0
+  elasticsearchRef:
+    name: elasticsearch-sample
+  # to target only a specific namespace for logging, use:
+  config:
+    filebeat.autodiscover.providers:
+    - node: ${NODE_NAME}
+      type: kubernetes
+      templates:
+      - config:
+        - paths: ["/var/log/containers/*${data.kubernetes.container.id}.log"]
+          type: container
```

Comment:
I don't get how this config targets a specific namespace?

Reply:
The config shouldn't be here at all, removing. I'll create samples for different setup separately.